setup.py

```diff
@@ -0,0 +1,32 @@
+# --------------------------------------------------------------------------
```

Comment:
Please remove this file from the PR. I have distribution code already, but I haven't yet included it in the Git repository. Issue #13 is intended to fix this soon.